arrow/cppbuild.sh

```diff
@@ -36,7 +36,7 @@ tar --totals -xzf ../openssl-$OPENSSL_VERSION.tar.gz
 tar --totals -xf ../llvm-$LLVM_VERSION.src.tar.xz || tar --totals -xf ../llvm-$LLVM_VERSION.src.tar.xz
 cd apache-arrow-$ARROW_VERSION
 patch -Np1 < ../../../arrow.patch
-sedinplace 's/ARROW_LLVM_VERSIONS "10"/ARROW_LLVM_VERSIONS "11" "10"/g' cpp/CMakeLists.txt
+sedinplace 's/ARROW_LLVM_VERSIONS "10"/ARROW_LLVM_VERSIONS "11" "10" "12"/g' cpp/CMakeLists.txt
```

Comment:
You probably got the order wrong here.

Reply:
OSX wouldn't build with the LLVM 12 version because that sed doesn't match the file contents anymore, they've put each version on a new line. Should have a fix for this locally which I will push.